Geometry_Engine/Query/Degree.cs

```diff
@@ -37,18 +37,5 @@ public static int Degree(this NurbsCurve curve)
         }
 
         /***************************************************/
-
-        public static List<int> Degrees(this NurbsSurface surf)
```

Comment:
Can we leave this method, deprecate it and make it return the `UDegree` and `VDegree` of the input `NurbsSurface`?

Nothing wrong about removing it, it's just to give some buffer to people that might have used it.
We can then remove it in the next beta

Reply:
You are right - actually I am not allowed to remove the method (can only deprecate it). I will do the latter and do my best to remove all references to it.